src/raw.rs

```diff
@@ -244,10 +246,29 @@ pub fn open_key(
 }
 
 #[allow(clippy::not_unsafe_ptr_arg_deref)]
+#[inline]
 pub fn reply_with_array(ctx: *mut RedisModuleCtx, len: c_long) -> Status {
     unsafe { RedisModule_ReplyWithArray.unwrap()(ctx, len).into() }
 }
 
+#[allow(clippy::not_unsafe_ptr_arg_deref)]
+#[inline]
+pub fn reply_with_map(ctx: *mut RedisModuleCtx, len: c_long) -> Status {
+    unsafe { RedisModule_ReplyWithMap.unwrap()(ctx, len).into() }
```

Comment:
Should we check if the API exists? and have a fallback?

Reply:
done